client/my-sites/checkout/checkout-thank-you/index.jsx

```diff
@@ -445,6 +446,12 @@ export class CheckoutThankYou extends React.Component {
 
 			return (
 				<Main className="checkout-thank-you">
+					{ this.props.transferComplete && (
+						<WpAdminAutoLogin
+							site={ { URL: `https://${ this.props.site?.wpcom_url }` } }
+							delay={ 0 }
```

Comment:
I've set the delay to 0 here in case the user clicks the button immediately. It's a lot of network traffic for a minute, but then the attempts should stop.